### PR TITLE
Fix for unicode urls

### PR DESCRIPTION
**Pull request by erikh360**
Unicode strings were not caught by my if statement `if isinstance(url, str):`. 

I'm now making sure the type is not list or tuple.

### CODE REVIEW

**Comment by sewagodimo** [comment]
Hey @erikh360 I am not exactly sure what the code you were testing for does so a description would be really helpful

File: message_sender/tests.py [hunk 1]
```diff
@@ -2062,6 +2062,46 @@ def test_send_voice_strip_filepath_language(self):
...
+            'message_delay': 10
+        }
+        channel = Channel.objects.create(**http_channel_override_payload)
+
+        message_sender = MessageClientFactory.create(channel)
+        res = message_sender.send_voice(
+            '+1234', '', speech_url=u'http://sbm.com/test.mp3',
+            session_event='new')
+
+        self.assertEqual(res['message_id'], 'message-uuid')
```
Comment (sewagodimo) [inline message_sender/tests.py:2098]: would it make a difference is the assert statements were consecutive?

Comment (erikh360) [reply message_sender/tests.py:2098]: If I understand correctly you mean if this assert moves down to the where the other one is, it won't really make a difference.

Comment (sewagodimo) [reply message_sender/tests.py:2098]: ooh yeah, I just thought they would look cuter all together at the bottom

**Review by sewagodimo — APPROVED** [review]

**Pull request merged**